gradio/components/model3d.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         """
         Parameters:
             value: path to (.obj, .glb, .stl, .gltf, .splat, or .ply) file to show in model3D viewer. If callable, the function will be called whenever the app loads to set the initial value of the component.
+            display_mode: the display mode of the 3D model in the scene. Can be "default", "point_cloud", or "wireframe".
```

Comment:
Maybe instead of "default", we call it "solid" for clarity? I'm not sure if that's the standard terminology (cc @dylanebert). If not, at least we should explain in the docstring:
```suggestion
            display_mode: the display mode of the 3D model in the scene. Can be "default" (which renders the model as a solid object), "point_cloud", or "wireframe".
```

Reply:
Just asked chatgpt, it agrees `solid` is better